maplibre/src/render/stages/upload_stage.rs

```diff
@@ -223,6 +238,66 @@ impl UploadStage {
                                         &feature_metadata,
                                     );
                                 }
+                                StoredLayer::RasterLayer {
```

Comment:
I would favor to create a new stage: upload_raster_stage and then conditionally add stat stage, instead of having the inline cfg flag